crypto/account.go

```diff
@@ -187,23 +187,10 @@ type LogicSigAccount struct {
 	SigningKey ed25519.PublicKey `codec:"sigkey"`
 }
 
-// MakeLogicSigAccountEscrow creates a new escrow LogicSigAccount. The address
-// of this account will be a hash of its program.
-// Deprecated: This method is deprecated for not applying basic sanity check over program bytes,
-// use `MakeLogicSigAccountEscrowChecked` instead.
-func MakeLogicSigAccountEscrow(program []byte, args [][]byte) LogicSigAccount {
-	return LogicSigAccount{
-		Lsig: types.LogicSig{
-			Logic: program,
-			Args:  args,
-		},
-	}
-}
-
-// MakeLogicSigAccountEscrowChecked creates a new escrow LogicSigAccount.
+// MakeLogicSigAccountEscrow creates a new escrow LogicSigAccount.
```

Comment:
@algochoi Prior to merge, can you add a CHANGELOG update?

The PR makes a couple of changes that I think ought to be called out.  I'm imagining it'll be easier to cover the bases prior to merge.  

Of course, if you prefer to address CHANGELOG later, feel welcomed to say so.

Reply:
Added in [8bcf6a2](https://github.com/algorand/go-algorand-sdk/pull/450/commits/8bcf6a284a5d06351dbc0c5c7c1475692b0f8eb3)